### DISCUSSION TITLE
QTable columns lose formatting when I use Body slot, is the correct behavior?

### DISCUSSION THREAD

**Discussion by jeangomes** [quasarframework/quasar#16113] [General - Components / Directives / etc]
I have some data formatting and style configured in the column definition.

Column example: 
`  {
    name: 'scheduled_date',
    required: true,
    label: 'Data',
    align: 'left',
    field: 'scheduled_date',
    format: val => getDate(val) + ', ' + getWeekDay(val),
    classes: row => getClasses(row.scheduled_date)
  }`

When I use Body Slot (to customize the entire row), format is not applied.
Is it really like that, I need call my logic on template?

**Comment by metalsadman** [comment]
assign in your qtd the class that's in the scope slot property ie. `props` like this `:class="props.__trClass"`, forgot the exact property, you can print the props to see it {{ props }} in the template.
